### PR TITLE
tests: Removed import *

### LINKED ISSUE #1125: Remove star imports
Replace the following `*` imports with explicit imports, to help prepare for https://github.com/coala/coala/issues/3066

```
tests/BearTestHelper.py:6: 'from coalib.testing.BearTestHelper import *' used; unable to detect undefined names
tests/BearTestHelper.py:6: 'coalib.testing.BearTestHelper.*' imported but unused
tests/LocalBearTestHelper.py:6: 'from coalib.testing.LocalBearTestHelper import *' used; unable to detect undefined names
tests/LocalBearTestHelper.py:6: 'coalib.testing.LocalBearTestHelper.*' imported but unused
```

### PR DESCRIPTION
**Pull request by vijeth-aradhya**
Removes the import *
and only added required dependencies
from LocalBearTestHelper.py and
BearTestHelper.py

Fixes #1125

### CODE REVIEW

**Comment by vijeth-aradhya** [comment]
@jayvdb Can you check why CircleCI is failing? I'll also have a look at it.

**Comment by Nosferatul** [comment]
Your shortlog is longer > 50. Please don't ping for such a thing. Also 'removed' word is bad

**Comment by vijeth-aradhya** [comment]
@Nosferatul I will change my commit message as per that, but was that the reason CircleCI was failing or was it because of the change in code? I'm sorry but I'm new to `Coala` and have to get acquainted with it :sweat_smile:

**Comment by jayvdb** [comment]
@vijeth-aradhya , please stop pinging people in every reply.  Focus on getting the checks passing, and come onto gitter if you need help or just want to chat. (and don't reply to this message ;-)

File: tests/BearTestHelper.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import logging
+from coalib.testing.BearTestHelper import generate_skip_decorator
```
Comment (nemaniarjun) [inline tests/BearTestHelper.py]: We may need a new line at `line: 2` according to PEP8.

File: tests/LocalBearTestHelper.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import logging
+from coalib.testing.LocalBearTestHelper import verify_local_bear
```
Comment (nemaniarjun) [inline tests/LocalBearTestHelper.py]: Same here

**Comment by nemaniarjun** [comment]
Your commit message should be in imperative mood. 

The message should be in form of "Removes import * ...."
instead of "I have removed... "

**Comment by nemaniarjun** [comment]
Looks good, but the commits are not different enough to be considered as two changes. Squash the two commits into one and keep the commit message of the first one.

If you have no idea what squashing is, search on google or ask on gitter. :)

**Comment by Mixih** [comment]
Sorry, AppVeyor is acting up. We have contacted them and are awaiting a response. Please hang in there while we work to resolve these issues

**Comment by jayvdb** [comment]
per https://github.com/coala/coala-bears/issues/1125#issuecomment-269083488 , I believe we dont need this commit.  My many apologies for not noticing this earlier.

**Pull request closed without merging**